Makefile

```diff
@@ -48,5 +48,19 @@ e2e-test:
 	dep ensure -v
 	go test -v ./test/e2e/ -args -nfd.repo=$(IMAGE_REPO) -nfd.tag=$(IMAGE_TAG_NAME) -kubeconfig=$(KUBECONFIG)
 
+verify:	verify-gofmt
```

Comment:
Some details, but I think you should replace this tab by a space

Reply:
Done.